.env

```diff
@@ -0,0 +1,7 @@
+PORT = 5001
```

Comment:
never push env file file to repo, just create .env.example where field will be described as PORT =    ; DB_NAME=